### ISSUE TITLE
SEO: google parses angular replacements

### ISSUE THREAD

**Issue by gjn** [geoadmin/mf-geoadmin3#2543]
From @davidoesch 
```
https://twitter.com/xmerour/status/626680875556700160/photo/1 
recht hat er, auch bei links in facebook erscheint oft gig_logo_small_logo
any idea on how to solve that?
```

Apparently, the idea that google executes javascript is not really true, or not true all the time. Otherwise, this wont' happen.

Long term solution is server-side rendering of pages, with fully translated content.

Short term: Maybe use english text as keys?

@oterral ?

**Comment by oterral** [comment]
Maybe google has saved a page during a deploy so the page has failed to load during a small amount of time.

**Comment by oterral** [comment]
We could use english as key but it could be diffuclt to maintain.

**Comment by oterral** [comment]
Have we tried this : https://developers.google.com/webmasters/ajax-crawling/docs/getting-started#3-handle-pages-without-hash-fragments ?

**Comment by gjn** [comment]
Yes, we have this: https://github.com/geoadmin/mf-geoadmin3/blob/master/src/index.mako.html#L26

**Comment by davidoesch** [comment]
ping @gjn

**Comment by boecklic** [comment]
@boecklic commented: Maybe google has saved a page during a deploy so the page has failed to load during a small amount of time.

**Comment by boecklic** [comment]
@boecklic commented: We could use english as key but it could be difficult to maintain.

**Comment by boecklic** [comment]
@boecklic commented: Have we tried this : https://developers.google.com/webmasters/ajax-crawling/docs/getting-started#3-handle-pages-without-hash-fragments ?

**Comment by boecklic** [comment]
@boecklic commented: Yes, we have this: https://github.com/geoadmin/mf-geoadmin3/blob/master/src/index.mako.html#L26

**Comment by boecklic** [comment]
@boecklic commented: ping @gjn
